scorer/verifications/brightid.py

```diff
@@ -0,0 +1,26 @@
+import time
+from arango import ArangoClient
+
+
+def verify():
+    print('BrightID')
+    db = ArangoClient().db('_system')
+    for u in db['users']:
+        verifications = set([v['name'] for v in db['verifications'].find({'user': u['_id']})])
+        callJoined = 'CallJoined' in verifications
+        seedConnected = 'SeedConnected' in verifications
+        if not (u['score'] >= 3 or (callJoined and seedConnected)):
```

Comment:
Now that we have a `verifications` collection that can store meta-data about a verification, it seems like this would be the natural place to store a user's score under a particular analysis method (i.e. verification).  In other words, `score` could be an attribute in `verifications`.  If a user has been evaluated by two different analyses (say `BrightID-A` and `BrightID-B`), then there could be
```
...
{
  "name": "BrightID-A",
  "user": "abc",
  "score": 25.2
},
{
  "name": "BrightID-B",
  "user": "abc",
  "score": 5.6
}
...
```
for example.  This is especially true since we no longer have a reason to return "score" as part of the "user" object in `/users`.  Instead we will return an array of verifications. See https://github.com/BrightID/BrightID-Node/issues/121

Reply:
I agree, but I think we can leave that to next release (`5.2`)